app/experimenter/nimbus-ui/README.md

```diff
@@ -51,7 +51,7 @@ You can read up over on their page if you want to know all the details, but if y
 
 All of our GraphQL resolvers produce types that are compatible with TypeScript, so there is no need to write new types for the queries you perform with Apollo.
 
-To generate types, just run `yarn types`. This will generate type declarations inside a `types/` directory at the root of this app. Once generated you can import and use as you see fit.
+To generate types, just run `yarn generate-types`. This will generate type declarations inside a `types/` directory at the root of this app. Once generated you can import and use as you see fit.  If there are changes to the graphql API on the server, run `make generate_types` to export the server schema and update the typescript stubs.
```

Comment:
Oh hah, I was actually referring to the bit a little further down about your local server needing to be running: 

> this command performs introspection on the GraphQL endpoint, so your local server must be running for it to work.

I don't want you to have to rerun CI just for that tho. Let's just get it in a later PR

Reply:
Oh no I gotta repush anyways, updated 👍

Reply:
Soooo happy about this change ahah 🙂

Reply:
Not needing to introspect a live server is a world of improvement for me.

Reply:
Yeah so glad I realized I could just dump the schema out from the server instead XD  That's how we do it for the old API and I realized wait I'll bet graphene does something similar.  Of course it does ;D